CHANGELOG.md

```diff
@@ -12,10 +12,13 @@ the code was deployed.
 ## [Unreleased]
```

Comment:
Reminder to self that I still need to test this on dev, but need to wait until I'm able to update `brave-alert-lib`.

Reply:
the tls fix and brave-alert-lib fix is now merged. Remember that this new stuff is all happening on the new cluster, which you can interact with from sensors-admin.brave.coop

Reply:
@mariocimet This is now up on `dev.sensors.brave.coop`. 
Let me know what you think :smile: